kombu/transport/redis.py

```diff
@@ -1066,14 +1066,17 @@ class SentinelChannel(Channel):
     from_transport_options = Channel.from_transport_options + (
         'master_name',
         'min_other_sentinels',
-        'sentinel_kwargs')
+        'sentinel_kwargs',
+        'connection_kwargs')
 
     connection_class = sentinel.SentinelManagedConnection if sentinel else None
 
     def _sentinel_managed_pool(self, async=False):
         connparams = self._connparams(async)
 
-        additional_params = connparams.copy()
+        additional_params = getattr(self,
+                                    'connection_kwargs',
+                                    connparams.copy())
```

Comment:
OK, then perhaps should be clarified in the [documentation](https://github.com/celery/celery/blob/master/docs/userguide/configuration.rst#redis-backend-settings)?